File: 2.0/java/numbers/findNumberLeaseConfigs.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.Page;
import com.callfire.api.client.api.numbers.model.NumberConfig;
import com.callfire.api.client.api.numbers.model.request.FindNumberLeaseConfigsRequest;

class ApiClientSample {
    public static void main(String[] args) {
        FindNumberLeaseConfigsRequest request = FindNumberLeaseConfigsRequest.create()
            .prefix("14245")
            .city("Los Angeles")
            .state("CA")
            .zipcode("90940")
            .lata("123")
            .rateCenter("123")
            .labelName("my label")
            .offset(0L)
            .limit(10L)
            .fields("items(number,configType)")
            .build();
        CallfireClient client = new CallfireClient("api_login", "api_password");
        Page<NumberConfig> configs = client.numberLeasesApi().findConfigs(request);
    }
}

File: 2.0/csharp/numbers/findNumberLeaseConfigs.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Numbers.Model;
using CallfireApiClient.Api.Numbers.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new FindNumberLeaseConfigsRequest()
        {
            Prefix = "14245",
            City = "Los Angeles",
            State = "CA",
            Zipcode = "90940",
            Lata = "123",
            RateCenter = "123",
            LabelName = "my label",
            Offset = 0,
            Limit = 10,
            Fields = "items(number,configType)"
        };
        Page<NumberConfig> configs = client.NumberLeasesApi.FindConfigs(request);
    }
}
